src/main/webapp/WEB-INF/tags/ksr/securityRequestRole.tag

```diff
@@ -6,16 +6,21 @@
               description="Whether the Security Request Role data should be read-only" %>
 
 <c:set var="genericAttributes" value="${DataDictionary.AttributeReferenceDummy.attributes}" />
-
 <c:set var="securityRequestRole" value="${KualiForm.document.securityRequestRoles[securityRequestRoleIndex]}" />
+<c:set var="roleTitle" value="${securityRequestRole.roleInfo.id} : ${securityRequestRole.roleInfo.namespaceCode} - ${securityRequestRole.roleInfo.name}"/>
 
-<h3>${securityRequestRole.roleInfo.id} : ${securityRequestRole.roleInfo.namespaceCode} - ${securityRequestRole.roleInfo.name}</h3>
+<h3>&nbsp;</h3>

```

Comment:
Do you still need empty horizonal or vertical space here? If so, could you use something simpler, such as a "br" tag?

Reply:
the h3 tag creates a horizontal link on the page, so this was added to help the users visually group distinct permission sections

Reply:
Thanks for the explanation. This should be fine as-is.